db/migrate/20160117211707_add_username_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :username, :string, null: false
```

Comment:
Yep, I ran into that problem locally and dropped the DB. Since this is a very early commit, I'm also fine just dropping the DB.

Sent from my iPhone

> On Jan 20, 2016, at 3:18 AM, Adarsh Pandit <notifications@github.com> wrote:
> 
> In db/migrate/20160117211707_add_username_to_users.rb:
> 
> > @@ -0,0 +1,6 @@
> > +class AddUsernameToUsers < ActiveRecord::Migration
> > +  def change
> > +    add_column :users, :username, :string, null: false
> What about existing users wit no username?
> 
> Note: I'm good with just dropping the production db at this point, despite the bad-practiceness of it all.
> 
> —
> Reply to this email directly or view it on GitHub.
>